examples/reference_plants/05-offshore-hydrogen-ca/input/plant/orbit-config-osw_15MW-ca.yaml

```diff
@@ -64,7 +64,7 @@ design_phases:
 - SemiSubmersibleDesign # from ORBIT/examples/configs/example_floating_project.yaml
 - ElectricalDesign  # new addition from Sophie Bradenkamp combined ExportSystemDesign and OffshoreSubstationDesign phases
 # - CustomArraySystemDesign
-- SemiTautMooringSystemDesign  # MooringSystemDesign old system design could use for very deep water but out of date design
+- MooringSystemDesign  # MooringSystemDesign old system design could use for very deep water but out of date design
```

Comment:
Is this comment still applicable?

Reply:
I'll double check on that and merge once I've got confirmation, but I think it's likely a dated comment at this point.